### PR TITLE
Issues 12 sivak

### PR DESCRIPTION
**Pull request by SpaceIgor**
#39 
-add command for driver /status_car - driver can change status car
-added command for service station manager /send report - manager can save report to db and send report driver "Your car <numberplace> renovated "
-add pytest for model service station
#43
-update view  /status for driver

### CODE REVIEW

**Comment by SpaceIgor** [comment]
@SergeyKutsko check

File: app/models.py [hunk 1]
```diff
@@ -592,6 +614,62 @@ def save_transactions_to_db(file_name):
                         print(f"Transaction is already in DB")
 
 
+class Car(models.Model):
```
Comment (SergeyKutsko) [inline app/models.py:617]: @SpaceIgor we already have model: Venicle

Comment (SpaceIgor) [reply app/models.py:617]: > @SpaceIgor we already have model: Venicle
my bad, our actions?

**Review by SergeyKutsko — APPROVED** [review]

**Comment by SpaceIgor** [comment]
@SergeyKutsko fixed conflict done

**Pull request merged**